install/install.sh

```
@@ -203,6 +211,20 @@ eth0_config(){
 	sudo nmcli connection down "Wired connection 1" && sudo nmcli connection up "Wired connection 1"
 }
 
+loveletter_extraction(){
+	sudo hostnamectl set-hostname $Boat
+	clone_repo $extract_tag $repo_extraction
+	cd LoveLetterExtraction
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)